app/src/main/java/com/limelight/computers/ComputerDatabaseManager.java

```diff
@@ -9,7 +9,9 @@
 import java.util.List;
```

Comment:
@kmreisi May I know the purpose of these changes? 
I have a problem with changes related to ComputerDatabaseManager.java & ComputerDetails.java
I connected to my PC from my phone remotely from WAN successfully but when I quit the app and opened the app again. The app only remembered the local ip of the PC but not the DDNS domain (which is required for connecting remotely).
![port open remote issue](https://user-images.githubusercontent.com/5093683/213912693-bae47d41-b634-4157-96c7-a1581b9a31fa.jpg)

I tried to keep ComputerDatabaseManager.java & ComputerDetails.java unchanged (based on master branch) and then it worked well again.
![port open remote without issue](https://user-images.githubusercontent.com/5093683/213912776-b80a6288-277c-4774-8cdd-2e3bcc4d7dee.jpg)

Moreover, removing changes from ComputerDatabaseManager.java & ComputerDetails.java didn't affect back menu functions.

Reply:
Hi @dangnm correct the ComputerDatabaseManager change is not related to the game menu. This is why there is a second PR https://github.com/moonlight-stream/moonlight-android/pull/1169. I just have based the game menu on top of the "fix" to allow me development on the branch. Since the game menu seems to be functional now I removed the change from this branch. :-)

I implemented the database fix because my host has an underscore in its name which does not work on current master. Based on your feedback I fixed https://github.com/moonlight-stream/moonlight-android/pull/1169. @dangnm Can you please try the changes again and report to the other PR. Thanks :-).

Reply:
Nice. let's discuss about the underscore issue in https://github.com/moonlight-stream/moonlight-android/pull/1169